### ISSUE TITLE
Black screen crash after login

### ISSUE THREAD

**Issue by GitesHubisz** [prasanthrangan/hyprdots#132]
Hello.

I apologize in advance if structure of this issue is poor. This is the first time I'm filing one on GitHub.

The problem I encountered is that when I try to your hyprdots on my old test machine (it's 12 years old, to be specific) and newly installed Arch, multiple issue related to the installed packages pop up. In most cases the server either went 504 (just like in the issue #122) or the installer found conflicts with the packages (like in the issue #129, I think). Also, as I'm using WiFi, some packages can't download or compile (the server refuses connection or there's a failure when importing PGP keys, as far as I can tell). I tried to bypass the problem by changing packages in 'custom_hypr.lst':
1. 'wlogout' to 'wlogout-git';
2. 'nwg-look' to 'nwg-look-bin';
3. 'hyprland-git' to 'hyprland'.

However, the server connects when I use an Ethernet cable and revert the packages to the default setup. The system reboots succesfully, shows my customized Grub theme, and SDDM works as well. Even so, when I try to log in, I get black screen with a mouse cursor stuck in the middle. I can't move it and it just hangs like that for hours. Luckily, I can still launch the terminal, so it's probably something to do with Window Manager, Still, the wireless connection don't work, but it's a minor inconvenience for me.

My main questions are: what may cause these problems, what are some things I can do to fix them and where can I find the necessary logs should you wish to see them?

Thank you in advance.

**Comment by jhenrique04** [comment]
Did you tried pacman-key --init and pacman-key --populate archlinux before installing arch? That should solve the issue with PGP keys

**Comment by jhenrique04** [comment]
Did you tried pacman-key --init and pacman-key --populate archlinux before installing arch? That should solve the issue with PGP keys

**Comment by GitesHubisz** [comment]
> Did you tried `pacman-key --init` and `pacman-key --populate archlinux` before installing arch (before archinstall or installing it manually)? That should solve the issue with PGP keys

No, but thanks for the tip. I will do that and if I'm gonna start it all over again.

**Comment by GitesHubisz** [comment]
Ok, I should write an update. First thing with `pacman-key --init` and `pacman-key --populate archlinux` I tried what jhenrique04 said, however, after trying to install packages, but I did not give anything (I will not describe how it did not work because it took a little time and I do not remember what exactly went). 

Second, I finally found the solution to my problem. In the OP, I wrote that my machine is very old. I looked on hyprland Wiki and found out about the Legacyrenderer. After going to ```~/.cache/yay/hyprland-git/src/Hyprland``` i follwed what the wiki said about this, and after this command:
```sudo make clear && sudo make config && make protocols && make legacyrenderer && sudo cp ./build/Hyprland /usr/bin && sudo cp ./example/hyprland.desktop /usr/share/wayland-sessions``` and using the installation script again, it worked, so now I can close this thread.
